src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php

```diff
@@ -474,6 +473,13 @@ private function createFirewall(ContainerBuilder $container, string $id, array $
                     ->addTag('kernel.event_subscriber', ['dispatcher' => $firewallEventDispatcherId]);
             }
 
+            // add clear site data listener
+            if ($firewall['logout']['clear_site_data'] ?? false) {
+                $container->setDefinition('security.logout.listener.clear_site_data.'.$id, new ChildDefinition('security.logout.listener.clear_site_data'))
+                    ->addArgument(implode(', ', array_map(fn ($v) => '"'.$v.'"', $firewall['logout']['clear_site_data'])))
```

Comment:
processing values at compile time is a bad practice; e.g. it breaks on env vars.
I would pass the array as is here

Reply:
Ok